hadoop-hdds/common/src/main/java/org/apache/hadoop/hdds/scm/net/NodeSchemaManager.java

```diff
@@ -76,6 +76,21 @@ public void init(ConfigurationSource conf) {
     }
   }
 
+  public void init(String schemaFile) {
```

Comment:
Could you refactor the init(ConfigurationSource conf) so that the two init will share the most common load logic.

Reply:
Refactored the `init` methods to share the common code, thanks!